hassio/src/dashboard/hassio-addons.ts

```diff
@@ -52,22 +52,22 @@ class HassioAddons extends LitElement {
                           .title=${addon.name}
                           .description=${addon.description}
                           available
-                          .showTopbar=${addon.installed !== addon.version}
+                          .showTopbar=${addon.installed ||
+                          addon.update_available!}
```

Comment:
This is a different logic than before, intended?

Reply:
No